src/main/universal/res/config/prelude_C.sil

```diff
@@ -58,11 +58,11 @@ domain VCTOption[T] {
   }
 
   axiom get_axiom_vct {
-    forall x: T :: getVCTOption(VCTSome(x)) == x
+    forall x: T :: {VCTSome(x)} getVCTOption(VCTSome(x)) == x
```

Comment:
Why is the trigger here not "getVCTOption(VCTSome(x))"?

Reply:
That indeed makes more sense I think, though triggering on VCTSome is harmless (there is no potential for matching loops here)